<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StrMappingDialog</class>
 <widget class="QDialog" name="StrMappingDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Mapping Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Source:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lw_src"/>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_auto_tgt_sel">
         <property name="text">
          <string>Auto target selection</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Target:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lw_tgt"/>
       </item>
       <item>
        <widget class="QCheckBox" name="chk_auto_src_sel">
         <property name="text">
          <string>Auto source selection</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>18</number>
       </property>
       <property name="rightMargin">
        <number>18</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="btn_add">
         <property name="text">
          <string>Add</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/right.svg</normaloff>:/images/right.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_delete">
         <property name="text">
          <string>Delete</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/left.svg</normaloff>:/images/left.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_auto">
         <property name="text">
          <string>Auto Mapping</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/right2.svg</normaloff>:/images/right2.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_clear">
         <property name="text">
          <string>Clear</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/images/left2.svg</normaloff>:/images/left2.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Mappings:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="lw_mapping"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_auto_2">
       <property name="text">
        <string>Auto Mapping</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_ok</tabstop>
  <tabstop>btn_cancel</tabstop>
  <tabstop>lw_tgt</tabstop>
  <tabstop>chk_auto_src_sel</tabstop>
  <tabstop>lw_mapping</tabstop>
  <tabstop>lw_src</tabstop>
  <tabstop>chk_auto_tgt_sel</tabstop>
  <tabstop>btn_add</tabstop>
  <tabstop>btn_delete</tabstop>
  <tabstop>btn_auto</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>btn_auto_2</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
